pkg/agent/datamodel/sig_config.go

```diff
@@ -348,6 +359,22 @@ var (
 		Version:       LinuxSIGImageVersion,
 	}
 
+	// This image is using a specific resource group and gallery name for edge zone scenario.
+	SIGUbuntuEdgeZoneContainerd1804ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSUbuntuEdgeZoneResourceGroup,
+		Gallery:       AKSUbuntuEdgeZoneGalleryName,
+		Definition:    "1804containerd",
+		Version:       EdgeZoneSIGImageVersion,
```

Comment:
this is fine but note that it means edge zones won't follow weekly process

possibly/probably they should? we can sync them manually once